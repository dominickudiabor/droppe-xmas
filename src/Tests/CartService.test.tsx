import React from 'react'
import renderer from 'react-test-renderer'
import Page from '../components/Page'

it('it matches snapshot', () => {
  const testPage = renderer
    .create(
      <Page header="save me">
        <p>Works well</p>
      </Page>
    )
    .toJSON()

  expect(testPage).toMatchSnapshot()
})
