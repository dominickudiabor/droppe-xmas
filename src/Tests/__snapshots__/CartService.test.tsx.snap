// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it matches snapshot 1`] = `
<div
  className="page"
>
  <div
    className="header"
  >
    <h2>
      save me
    </h2>
  </div>
  <p>
    Works well
  </p>
</div>
`;
